src/v2/Apps/Auction/Routes/__tests__/ConfirmBid.jest.tsx

```diff
@@ -350,8 +377,8 @@ describe("Routes/ConfirmBid", () => {
       await page.submitForm()
 
       expect(window.location.assign).not.toHaveBeenCalled()
-      expect(page.text()).toContain(
-        "Please make sure your internet connection is active and try again"
+      expect(page.find(ErrorModal).text()).toContain(
+        "Something went wrong. Please try again"
```

Comment:
Where is this error message coming? The fallback error message text? https://github.com/artsy/force/blob/be156e9feda1505b3a231e6c3ea347d1a1a02cb9/src/v2/Components/Modal/ErrorModal.tsx#L54

Reply:
>  The fallback error message text?

Correct.